pkg/kubectl/cmd/scale.go

```diff
@@ -146,6 +147,10 @@ func RunScale(f cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []strin
 		}
 
 		mapping := info.ResourceMapping()
+		if mapping.Resource == "jobs" {
+			glog.Warningf("%s scale job is DEPRECATED and will be removed in a future version.", cmd.Parent().Name())
```

Comment:
we normally use `fmt.fprintf(stderr` to avoid irrelevant line numbers.

Reply:
That's what I had originally, but then I found [this cherry](https://github.com/kubernetes/kubernetes/blob/ecc5eb67d965295db95ba2df5f3d3ff43a258a05/pkg/kubectl/cmd/cmd.go#L358-L360). Although I'm fine fixing the other one as well to use `fmt.Fprintf`